explainer.md

```diff
@@ -77,16 +73,24 @@ interface MemoryInfo {
   // All memory used by the JS heap in the process hosting the site.
   // Includes everything from usedJSHeapSize, and also fragmentation.
   readonly attribute unsigned long? totalJSHeapSize;
+
+  // IMPORTANT: When this number is greater than 1, all metrics over count by an
+  // undefined amount. It's recommended for most developers to simply ignore the
+  // results of Performance.Memory in this case.
+  //
+  // This number describes the number of frame trees with at least one frame
```

Comment:
I am wondering what the right terminology here is... maybe it would be better to talk about tabs and not about frame trees.

Reply:
Done